package.json

```diff
@@ -43,6 +43,7 @@
     "eslint-plugin-import": "^2.26.0",
     "eslint-plugin-matrix-org": "^0.4.0",
     "jest": "^27.5.1",
+    "react": "17.0.2",
```

Comment:
React is used under `src` → dev dependencies is the wrong place.
I would suggest to move it to `peerDependencies`.

Reply:
changed that, thanks for notice

Reply:
If this change is applied, then `BannerLifecycle.test.ts` is failing with: Cannot find module 'react', so it is used in tests. So reverted the change.

Reply:
Probably it should then be both: A dev and a peer dependency.